src/main/java/io/github/thebusybiscuit/sensibletoolbox/items/EnderLeash.java

```diff
@@ -193,8 +202,9 @@ private YamlConfiguration freezeEntity(Animals target) {
             conf.set("sheared", ((Sheep) target).isSheared());
             conf.set("color", ((Sheep) target).getColor().toString());
             break;
-        case OCELOT:
-            conf.set("catType", ((Ocelot) target).getCatType().toString());
+        case CAT:
+            conf.set("catType", ((Cat) target).getCatType().toString());
+            conf.set("setCollarColor", ((Cat) target).getCollarColor().toString());
```

Comment:
ehh why "set" ?

Reply:
That was a silly typo.. which lead me to find another, glad you caught that.